heron/spi/src/java/com/twitter/heron/spi/common/keys.yaml

```diff
@@ -55,6 +55,7 @@ STATE_MANAGER_CLASS:                 heron.class.state.manager
 # keys for scheduler config
 ################################################################################
 SCHEDULER_IS_SERVICE:                heron.scheduler.is.service
+HERON_SCHEDULER_YARN_QUEUE:          heron.scheduler.yarn.queue
```

Comment:
Would it be useful to update the default value in the `yarn/scheduler.yaml` too. When needed users could find it in the config file and avoid searching for it. Also could you please add this config in the document.